templates/repo/commit_page.tmpl

```diff
@@ -83,7 +83,7 @@
 													<input id="new_branch_name" name="new_branch_name" required>
 												</div>
 
-												<div class="text right actions">
+												<div class="tw-text-right actions">
```

Comment:
All `actions` for modals, could(should) only keep `actions`.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/33905/commits/38f4214a9fb034915fc0cc35f9d31e429aea987e and https://github.com/go-gitea/gitea/pull/33905/commits/b90fcd3f62777b1e6e03a5b350d7f227d63195fc.

There seems to be some needless complexity because some modals (like create branch on commit page) do not match the existing `.ui .modal > .actions` because they have `.modal > form > .actions`, so I decided to play it safe and create another selector.

Reply:
Is it this case? I do not think we should create a separate selector. Improving existing selectors should be the right thing to do.

![image](https://github.com/user-attachments/assets/bc2a3924-7a95-4522-8454-75160ba5c286)

Reply:
I agree to changing to `.modal .actions` selector, but at least the `padding` added needs to go I think.

Reply:
6b9c0c0dd42440629daca28eaba0a57b9458f092

Reply:
Works, thought I'm not super happy with another selector, but ok.

Reply:
> I agree to changing to selector without `>`, but at least the `padding` added needs to go I think, and also the `background`.

We need to make it strict here, because there are different styles (eg: padding, background) for different "actions", so we need to define each case explicitly (see the devtest page)